### ISSUE TITLE
OneeChan hide 4chan ads, whether "Show 4chan Ads" is checked or not

### ISSUE THREAD

**Issue by hellotinh03** [KevinParnell/OneeChan#43]
Title said it all.
<img width="1440" alt="Ảnh chụp Màn hình 2021-09-09 lúc 17 24 05" src="https://user-images.githubusercontent.com/47333723/132669399-1e42f8d9-1578-45c5-ab7c-9a0638aa5707.png">

**Comment by KevinParnell** [comment]
Are you using an adblocker?

**Comment by hellotinh03** [comment]
> Are you using an adblocker?

Yes I'm using uBlock, but the issue only happens when Oneechan is being used.
